From cc9c7677e66598ed2e93306a747253b2a5a9ec27 Mon Sep 17 00:00:00 2001
From: John MacFarlane <jgm@berkeley.edu>
Date: Tue, 22 Sep 2015 22:40:12 -0700
Subject: [PATCH] Set convert_charrefs=False in normalize.py.

This defeats the new default as of python 3.5, and allows
the script to work with python 3.5.

Closes #83.
---
 test/normalize.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/normalize.py b/test/normalize.py
index 9025bfa..6eb4ec2 100644
--- a/test/normalize.py
+++ b/test/normalize.py
@@ -21,6 +21,7 @@ class HTMLParseError(Exception):
 class MyHTMLParser(HTMLParser):
     def __init__(self):
         HTMLParser.__init__(self)
+        self.convert_charrefs = False
         self.last = "starttag"
         self.in_pre = False
         self.output = ""
